src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -389,11 +377,13 @@ struct KernelFillGaps
 
             if ( counterGaps == 0 )
             {
-                updateCellsForOneFrame<TileSize>(cellEntryPoint_sh, cellCount_sh, firstFrame, frameNr);
+                if (updateCellList)
+                    kernel::UpdateCellListOfFrame{}.template operator()<TileSize>(*cellEntryPoint_sh, *cellCount_sh, firstFrame, *frameNr);
```

Comment:
yes, that's just wonderful C++ syntax in member functions with templates :)

you might know this style from things like our `map<...>` functions in kernels :)